programmers/K번-째-수.js

```diff
@@ -0,0 +1,15 @@
+// ����� �ٽ� ���� �����ϱ�.
+// �ִ��� ���� ��԰� �ٽ� Ǯ��.
+function solution(array, commands) {
+    const result = [];
+    for(let x = 0; x < commands.length; x += 1) {
+        let i = commands[x][0];
+        let j = commands[x][1];
+        let k = commands[x][2];
+        
+        const extRactArr = array.slice(i - 1, j)
+        const sortedArr = extRactArr.sort((a, b) => a < b ? -1 : 0);
+        result.push(sortedArr[k - 1])
+    }
+    return result
+}
```

Comment:
```suggestion
function solution(array, commands) {
    const result = [];
    
    for (let i = 0; i < commands.length; i += 1) {
        const first = commands[i][0];
        const end = commands[i][1];
        const target = commands[i][2];

        const sliceArr = array.slice(first - 1, end);
        const sortedArr = sliceArr.sort((a, b) => a - b);
        
        result.push(sortedArr[target - 1]);
    }
    return result;
}
```
변수 선언과 for문 시작 줄바꿈해주세요 ~
변수명에 i, j, k같은 변수는 사용하지마세요 주로 index를 표현할 때 사용합니다.
sort문에서는 삼항연산자 사용도 좋지만 간단하게 표현이 가능합니다.

변수명은 좀 더 알아보기 쉽게 지정해주세요